### ISSUE TITLE
Weight normalization during inference.

### ISSUE THREAD

**Issue by MohitLamba94** [JiahuiYu/wdsr_ntire2018#47]
Hello,

`torch.nn.utils.weight_norm` does the crucial weight re-parametrization during training, but does weight normalization(re-parametrization) also happen during testing? If yes should not they be removed before proceeding for testing? Or is it alright that weight normalization happen during testing also?

Thankyou

**Comment by JiahuiYu** [comment]
> is it alright that weight normalization happen during testing also?


Yes.
